packages/create-rspack/index.js

```diff
@@ -4,23 +4,29 @@ const { hideBin } = require("yargs/helpers");
 const fs = require("fs");
 const path = require("path");
 const prompts = require("prompts");
+const { formatTargetDir, emptyDir } = require("./utils/index");
 yargs(hideBin(process.argv))
 	.command("$0", "init rspack project", async argv => {
 		const defaultProjectName = "rspack-project";
-		let result = await prompts([
+		let targetDir = defaultProjectName;
+		await prompts([
 			{
 				type: "text",
 				name: "projectDir",
 				initial: defaultProjectName,
-				message: "Project folder"
+				message: "Project folder",
+				onState: state => {
+					targetDir = formatTargetDir(state.value) || defaultTargetDir;
+				}
 			}
 		]);
-		const { projectDir } = result;
-		const root = path.resolve(process.cwd(), projectDir);
+		const root = path.resolve(process.cwd(), targetDir);
 		if (fs.existsSync(root)) {
-			throw new Error("project directory already exists");
+			emptyDir(root);
```

Comment:
I guess we're not supposed to delete the user directory by accident, so it will be better to prompt up and give users other shots until there are no conflicts.

Reply:
Yes，I will optimize it.Thank you

Reply:
done